src/cargo/util/toml.rs

```diff
@@ -144,11 +144,6 @@ pub fn to_manifest(contents: &[u8],
                     })
                 }
             }
-            toml::Value::Array(ref arr) => {
```

Comment:
I'm not entirely sure why this existed in the first place. Let me know if I'm overlooking something here